File: Source/Back-end/src/main/java/org/elis/manutenzione/dto/response/GetMessaggioResponse.java
package org.elis.manutenzione.dto.response;

import lombok.Data;

/**
 * DTO per la risposta contenente i dati di un messaggio.
 * Include l'ID, il mittente e il testo del messaggio.
 */
@Data
public class GetMessaggioResponse {
    /**
     * L'ID univoco del messaggio.
     */
    private Long id;
    /**
     * L'email del mittente del messaggio.
     */
    private String mittente;
    /**
     * Il testo del messaggio.
     */
    private String testo;
}

File: Source/Front-end/front-end/src/app/model/response/get-messaggio-response.ts
export class GetMessaggioResponse {
  id: number;
  mittente: string;
  testo: string;

  constructor(id: number, mittente: string, testo: string) {
    this.id = id;
    this.mittente = mittente;
    this.testo = testo;
  }
}
